src/main/drivers/pwm_output.h

```diff
@@ -17,13 +17,18 @@
 
 #pragma once
 
+typedef enum {
+    PWM_TYPE_CONVENTIONAL = 0,
```

Comment:
Betaflight is now using `PWM_TYPE_STANDARD` rather than `PWM_TYPE_CONVENTIONAL`.